lib/internal/streams/duplex.js

```diff
@@ -63,6 +63,24 @@ function Duplex(options) {
   if (!(this instanceof Duplex))
     return new Duplex(options);
 
+  this._events ??= {
```

Comment:
I don't really think we should modify the private property `_events` like that, maybe have an option or an api for setting events template in the EventEmitter class?

Reply:
Follow up PR welcome?